### PR TITLE
Invoke onshippingcontactselected when "postalAddress" is passed in "requiredShippingContactFields"

### LINKED ISSUE #8: Invoke onshippingcontactselected when "postalAddress" is passed in "requiredShippingContactFields"
Hi and thanks for the great polyfill!

I figured out that currently ApplePay in browser invokes `onshippingcontactselected` only when  `postalAddress` is passed in `requiredShippingContactFields`. There is no check [here](https://github.com/justeat/applepayjs-polyfill/blob/master/ApplePaySession.js#L323), so I guess might me useful.

### PR DESCRIPTION
**Pull request by mariovalney**
Fixes #8

### CODE REVIEW

**Comment by mariovalney** [comment]
I'm not sure why the df4f9432d708e8df4084ce7906fa5ff721571fc8 commit was necessary.

I checked editorconfig and spaces. Any clues to recreate the PR without this?

**Comment by martincostello** [comment]
Looks like this repo doesn't actually have ~an `.editorconfig` file in it, nor~ a `.gitattributes`. I'll look to add an appropriate one ~of each~, then if you rebase once I've done that _hopefully_ your change should apply without all the whitespace changes.

**Comment by martincostello** [comment]
That's done. I had it do something weird just now with changes, but then it stopped doing it in Visual Studio Code. Now quite sure what's happening.

**Comment by mariovalney** [comment]
Rebased! Thanks :)

File: ApplePaySession.js [hunk 1]
```diff
@@ -337,13 +346,15 @@
...
+                var hasPostalAdress = self.paymentRequest && ("requiredShippingContactFields" in self.paymentRequest) && ("postalAddress" in self.paymentRequest.requiredShippingContactFields);
+
+                if (typeof session.onshippingcontactselected === "function" && hasPostalAdress) {
 
                     var applePayShippingContactSelectedEvent = {
                         shippingContact: self.createShippingContact(session)
                     };
 
-                    session.onshippingcontactselected(applePayShippingContactSelectedEvent);
+                    self.onShippingContactSelectedWithTimeout(session, applePayShippingContactSelectedEvent)
```
Comment (martincostello) [inline ApplePaySession.js]:
```suggestion
                    self.onShippingContactSelectedWithTimeout(session, applePayShippingContactSelectedEvent);
```

**Review by martincostello — APPROVED** [review]

**Comment by martincostello** [comment]
Thanks again 👍

**Pull request merged**